README.md

```diff
@@ -51,6 +51,7 @@ folder and try to start the server again)_
 ```bash
 # In the figwheel server REPL run:
 (redeploy)
+(generate-data)
```

Comment:
Aha, interesting. The section title indicates maybe this is the intended usage. I'd add an `;; and optionally` line above this, but I leave that up to you.

Reply:
Yeah, good idea.